api/lib/notifications.js

```diff
@@ -214,6 +222,10 @@ const startNotificationService = (networks) => {
     eventSubscription(async (event) => {
       if (event.properties.type === 'UnknownTx') return
 
+      // hack to not spam users on every liveness failure
+      // need to figure out how to handle this
+      if (event.eventType === eventTypes.LIVENESS) return
```

Comment:
so this happens a lot?

Reply:
it could happen every block. if you check the slashes table you see some liveness failures >2000 blocks